scripts/cli/index.ts

```diff
@@ -0,0 +1,54 @@
+// #!/usr/bin/env node
+import { Command } from 'commander'
+
+import { actions } from './actions'
```

Comment:
``` import { actions, generators } from './actions' ```

Reply:
It's not exported from this file :(

Reply:
I don't want to create any require changes since it's not showing nicely like in react native.

Reply:
My bad!